odoo/addons/base/models/assetsbundle.py

```diff
@@ -407,18 +407,31 @@ def get_template(element):
             string = etree.tostring(element, encoding='unicode')
             return string.replace("\\", "\\\\").replace("`", "\\`").replace("${", "\\${")
 
+        names = set()
+        inherit_froms = set()
         for block in blocks:
             if block["type"] == "templates":
-                for (element, url) in block["templates"]:
+                for (element, url, inherit_from) in block["templates"]:
+                    if inherit_from:
+                        inherit_froms.add(inherit_from)
                     name = element.get("t-name")
+                    names.add(name)
                     template = get_template(element)
                     content.append(f'registerTemplate("{name}", `{url}`, `{template}`);')
             else:
                 for inherit_from, elements in block["extensions"].items():
+                    inherit_froms.add(inherit_from)
                     for (element, url) in elements:
                         template = get_template(element)
                         content.append(f'registerTemplateExtension("{inherit_from}", `{url}`, `{template}`);')
 
+        missing = set()
+        for inherit_from in inherit_froms:
+            if inherit_from not in names:
+                missing.add(inherit_from)
```

Comment:
```suggestion
        missing = inherit_froms - names
```
minus on sets does set difference